src/apps/across/ethereum/eth.across.lp-fetcher.ts

```diff
@@ -0,0 +1,78 @@
+import { Inject } from '@nestjs/common';
+
+import { Register } from '~app-toolkit/decorators';
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { TvlFetcher } from '~stats/tvl/tvl-fetcher.interface';
+import { Network } from '~types/network.interface';
+
+import { AcrossContractFactory } from '../contracts';
+import ACROSS_DEFINITION from '../across.definition';
+
+const appId = ACROSS_DEFINITION.id;
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.TvlFetcher({ appId, network })
+export class EthereumAcrossTvlFetcher implements TvlFetcher {
```

Comment:
Can you rename this file: `src/apps/across/ethereum/eth.across.lp-fetcher.ts` to `src/apps/across/ethereum/across.tvl-fetcher.ts`

Also, if there are no balances to be reported for the users, please delete all of the `across.balance-fetcher.ts` files.